nixos/hardware/hetzner-online/default.nix

```diff
@@ -30,11 +30,9 @@
 
     networking.useNetworkd = true;
     networking.useDHCP = false;
-    # Hetzner servers commonly only have one interface, so its either to just match by that.
-    networking.usePredictableInterfaceNames = false;
 
     systemd.network.networks."10-uplink" = {
-      matchConfig.Name = "eth0";
+      matchConfig.Name = lib.mkDefault "eno1 ens1 enp1s0 eth0";
```

Comment:
> enp1s0

Upon revisiting this, i think this part might be to naive. Not sure if enp0s0 would be better, need to check docs again.

Reply:
https://www.freedesktop.org/software/systemd/man/systemd.net-naming-scheme.html

Reply:
My question above would only have been relevant if we needed a consistent definition of the "first" interface. We don't need that here, and so we can just match on all ethernet devices.